internal/controller/cluster_monitoring_controller.go

```diff
@@ -249,7 +249,7 @@ func (r *ClusterMonitoringReconciler) reconcile(ctx context.Context, cluster *cl
 		}
 	} else {
 		// clean up any existing prometheus agent configuration
-		err := r.PrometheusAgentService.DeleteRemoteWriteConfiguration(ctx, cluster)
```

Comment:
I would prefer that we move export the services instead because having them embedded make it hard to know which service I'm actually calling

Reply:
Oh ok I see